Makefile

```diff
@@ -240,6 +240,13 @@ clean_certs:
 ###                  Formatting, linting, and vetting                       ###
 ###############################################################################
 
+#? setup-pre-commit: Set pre-commit git hook
+setup-pre-commit:
```

Comment:
There's `gofumpt` pre-commit hook. Can we merge these two?

Reply:
Okay, I will merge them